README.md

```diff
@@ -246,6 +246,26 @@ for example. See [Creating & using secrets]. While still secure,
 [trusted publishing] is now encouraged over API tokens as a best practice
 on supported platforms (like GitHub).
 
+### Generating and uploading attestations (EXPERIMENTAL)
```

Comment:
I'm not sure if the ` (EXPERIMENTAL)` bit should be in the title, because removing it later will break old URLs from places where people (and ourselves) would link this title. Can we avoid this somehow?

Reply:
Yeah, good call -- there's already a NOTE callout right below, so I think just removing `(EXPERIMENTAL)` should be fine 🙂